src/gatsby-ssr.js

```diff
@@ -12,9 +12,10 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
     manualLoad,
     loadType,
     useNewSDK,
+    sdkURL,
   } = pluginOptions;
 
-  var sdkSrc = "https://cdn.rudderlabs.com/v1/rudder-analytics.min.js";
+  var sdkSrc = sdkURL || "https://cdn.rudderlabs.com/v1/rudder-analytics.min.js";
   if (useNewSDK) {
     sdkSrc = "https://cdn.rudderlabs.com/v1.1/rudder-analytics.min.js";
   }
```

Comment:
Same in here also. We can move this checking before line 18.

Reply:
I've addressed the logical error in the recent commit.